src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -148,7 +152,17 @@ def set_timesteps(self, num_inference_steps: int, device: Union[str, torch.devic
 
         timesteps = np.linspace(0, self.config.num_train_timesteps - 1, num_inference_steps, dtype=float)[::-1].copy()
         sigmas = np.array(((1 - self.alphas_cumprod) / self.alphas_cumprod) ** 0.5)
-        sigmas = np.interp(timesteps, np.arange(0, len(sigmas)), sigmas)
+
+        if interpolation_type == "linear":
```

Comment:
@patil-suraj @pcuenca 

What do you think about the API here? I now wonder whether it might be better to put `interpolation_type` in the config instead as it's not something one would like to change for the same scheduler no? 

Also @yiyixuxu @williamberman

Reply:
It sounds to me like it should be a part of the config.